cmd/trace/main.go

```diff
@@ -1,17 +1,12 @@
 package main
 
 import (
-	"fmt"
 	"os"
 
 	"github.com/xhd2015/xgo/cmd/xgo/trace"
 )
 
 func main() {
 	args := os.Args[1:]
-	err := trace.Main(args)
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "%v\n", err)
-		os.Exit(1)
-	}
+	trace.Main(args)
```

Comment:
May I suggest something?

Here and in previous code, you moved the `os.Exit(1)` inside the code, it's a bad pattern for me.

The code becomes almost impossible to test, because of the exit.

Maybe you made the change to be consistent with other part of code but I would have suggested to keep the error.

Why did you change ?

Also, this pattern leads to errors as os.Exit(1) can be forgotten

Reply:
I can help in refactoring it, and suggest you something if you agree.

You may accept it or not, but I would like to show what I have in mind.

One question, do you plan to support other exit codes than 1?